.github/workflows/main.yml

```diff
@@ -113,74 +112,57 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with: 
-          registry-url: ${{ env.GH_REGISTRY }}
-          scope: ${{ env.SCOPE }}
-      
-      - name: Update package.json to specify scope for ECAD
-        run: | 
-          export scope="${{ env.SCOPE }}/"
-          export name="$(jq -r .name package.json | awk '{split($0,a,"/"); print a[2]}')"
-          contents=$(jq '.name |= env.scope + env.name' package.json) && \
-          echo ${contents} > package.json
+          registry-url: ${{ env.NPM_REGISTRY }}
+          scope: "@taqueria"
+
+      - name: Check tag suffix
+        id: check-tag-suffix
+        if: startsWith(github.ref, 'refs/tags/v') && (contains(github.ref, '-rc') || contains(github.ref, '-beta') || contains(github.ref, '-test'))
+        run: |
+          echo "::set-output name=npm-tag::next"
 
       - name: Install any dependencies
-        run: npm install
+        run: npm ci
 
       - name: Build package
         run: npm run build
-
-      - name: Get current local package version
-        id: get-local-package-version
-        run: echo "::set-output name=version::$(jq .version package.json)"
       
-      - name: Get current npm registry version
-        id: get-npm-package-version
+      - name: Publish for PR
         env: 
-          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPMJS_PAT }}
+        if: ${{ github.event_name == 'pull_request' }}
         run: |
-          echo "$(jq -r .name package.json)"
-          echo "::set-output name=version::$(npm show $(jq -r .name package.json) --json | jq .version)"
+          npm --no-git-tag-version version "0.0.0-pr-${{ github.event.number }}-${GITHUB_SHA::8}"
+          npm publish --tag "pr-${{ github.event.number }}" --access public
       
-      - name: Publish if new version of package 
+      - name: Publish for prerelease
         env: 
-          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        if: steps.get-npm-package-version.outputs.version < steps.get-local-package-version.outputs.version
-        run: npm publish 
+          NODE_AUTH_TOKEN: ${{ secrets.NPMJS_PAT }}
```

Comment:
You can make this env variable available for the entire `job` if you intend to use it in more than one step

Reply:
I changed the env variable to the `job` scope